client-src/elements/chromedash-feature-table.ts

```diff
@@ -88,7 +86,7 @@ class ChromedashFeatureTable extends LitElement {
     window.csClient
       .getPendingGates()
       .then(res => {
```

Comment:
To find the right types for internal values, start by adding a type to the result of the API call. I think this one is

```suggestion
      .then((res: {gates: GateDict[]}) => {
```
But please double-check that.